automatic/intunewinapputil/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Win32 Content Prep Tool
+
+      Manage Windows Apps (.intunewin) with Intune
+
+      [See release notes for more information.](https://github.com/Microsoft/Microsoft-Win32-Content-Prep-Tool/releases)
+
+      Use the Microsoft Win32 Content Prep Tool to pre-process Windows Classic apps. The packaging tool converts application installation files into the .intunewin format. The packaging tool also detects the parameters required by Intune to determine the application installation state. After you use this tool on your apps, you will be able to upload and assign the apps in the Microsoft Intune console.
+
```

Comment:
Please include a `Features` section about the main features of the application.

Reply:
Added the Features heading here.